### PR TITLE
[Safe Creation] Owner step

### LINKED ISSUE #962: [Create Safe] Implement redesign Owner step
## What is the feature about

Implements the redesign for the owner step in safe creation.

## The list of requirements

Owner
- Its possible to add a name for an owner
- Its possible to import an owner via QR code
- Its possible to delete an owner
- Its possible to add more owners
- At least one owner must exist
- Displays a hint for an owner field if its the currently connected wallet

Mobile owner
- Its possible to add a safe mobile owner key
- Displays a hint on hover for safe mobile owner keys

Threshold
- Its possible to set the threshold
- The threshold field adjusts dynamically depending on the number of owners
- Displays a hint on hover for the threshold

## Designs/sketches
Screenshot 2022-10-24 at 12 01 00

## Links

Figma: https://www.figma.com/file/ptTs6lDBeUuLNySroJ5PiF/Web-Master-File?node-id=56%3A6575

### PR DESCRIPTION
**Pull request by schmanu**
## What it solves
Redesign of owner step.

Resolves #962 

## How this PR fixes it
Adds New owner and threshold step with redesigned elements.

## How to test it
Navigate to `/demo` and observe the example owner step.
The safe creation and the owner step will be fully functional once #931 is resolved.

## Analytics changes
None

## Screenshots
Screenshot 2022-10-26 at 17 48 19

### CODE REVIEW

File: src/components/new-safe/steps/Step2/OwnerRow.tsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  return (
+    <Grid container spacing={3} alignItems="flex-start" marginBottom={3} flexWrap={['wrap', undefined, 'nowrap']}>
+      <Grid item xs={12} md={4}>
+        <FormControl fullWidth>
+          <NameInput
+            name={`${fieldName}.name`}
+            label="Owner name"
+            InputLabelProps={{ shrink: true }}
+            placeholder={ens || `Owner ${index + 1}`}
+            helperText={owner.address === wallet?.address && 'Your connected wallet'}
```
[the remark this answers is not in the document]
Comment (schmanu) [reply src/components/new-safe/steps/Step2/OwnerRow.tsx:76]: Already addressed in the stepper branch.

**Review by usame-algan — APPROVED** [review]
🚀

File: src/components/new-safe/steps/Step2/OwnerRow.tsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+      }
+    },
+    [owners],
+  )
+
+  const { ens, name, resolving } = useAddressResolver(owner.address)
+
+  useEffect(() => {
+    if (ens) {
+      setValue(`${fieldName}.ens`, ens)
```
Comment (iamacook) [inline src/components/new-safe/steps/Step2/OwnerRow.tsx:59]: Why need we save the ENS?

Comment (iamacook) [inline src/components/new-safe/steps/Step2/OwnerRow.tsx:59]: Why need we save the ENS?

**Pull request merged**